rpc_jobs/rpc_maas.yml

```diff
@@ -84,9 +84,9 @@
     jobs:
       - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
 
-# NOTE(mattt): the rpc-maas team would rather run all functional
-#              tests on PR (`check`), rather than `gate`. Leaving these
-#              all commented for now.
+# TODO(mattt): We need to split out existing pre-merge jobs between check and
+#              gate, but this will be done after rpc-maas has been
+#              successfully added to the release pipeline.
```

Comment:
Even if gate is a no-op, we should still be creating a gate job, so people get used to @rpc-jenkins-svc doing the merges.

Reply:
@hughsaunders they can still run `(re)?check_component_gate` without there being any gate jobs.

Reply:
@hughsaunders You don't need any actual gate jobs defined for them to be able to run `(re)?check_component_gate` and have Jenkins handle the merge.